lib/vendors/axis-communications/sv_vendor_axis_communications_internal.h

```diff
@@ -153,7 +153,7 @@ get_axis_communications_trusted_certificate(void);
 /**
  * @brief Retrieves the certificate chain from the vendor handle.
  *
- * @param sv The signed_video_t structure containing vendor_handle.
+ * @param handle The handle containing certificate chain.
```

Comment:
I think you should use a description like on line 133. By design, the user already knows that the certificate chain is "inside" the handle, but what handle should be passed in. That is the kind of information to provide the user.